packages/web/src/common/store/pages/audio-rewards/sagas.ts

```diff
@@ -131,9 +131,15 @@ const getClaimingConfig = (
   remoteConfigInstance: RemoteConfigInstance,
   env: Env
 ) => {
-  const quorumSize = remoteConfigInstance.getRemoteVar(
-    IntKeys.ATTESTATION_QUORUM_SIZE
-  )
+  const { ENVIRONMENT } = env
+  let quorumSize
+  if (ENVIRONMENT === 'development') {
+    quorumSize = 2
```

Comment:
We'll want to watch out for this when/if we set the quorum size to 3. Unfortunate that we don't have better config around this here.